local_planner/test/test_local_planner.cpp

```diff
@@ -87,7 +87,7 @@ TEST_F(LocalPlannerTests, all_obstacles) {
 
   for (size_t i = 0; i < scan.ranges.size(); i++) {
     // width determined empirically, TODO investigate why it isn't symmetrical
```

Comment:
Does that take care of this TODO?

Reply:
Julian wrote the TODO and I'm not completely sure if that fixes what he meant.

Reply:
@jkflying

Reply:
Are they symmetrical now? We should be able to determine the indexes based on the geometry to the obstacles we generate.